packages/sitecore-jss/src/layout/rest-layout-service.ts

```diff
@@ -1,33 +1,114 @@
-import { AxiosDataFetcher, AxiosDataFetcherConfig } from './axios-fetcher';
-import { LayoutServiceData, PlaceholderData } from './data-models';
-import { fetchPlaceholderData, fetchRouteData, LayoutServiceConfig } from './data-api';
-import { GraphQLRequestClient } from './graphql-request-client';
-import { HttpDataFetcher } from './data-fetcher';
 import { AxiosRequestConfig, AxiosResponse } from 'axios';
-import { IncomingMessage, ServerResponse } from 'http';
-import debug from './debug';
+import { LayoutServiceBase, IncomingMessage, ServerResponse, models } from './layout-service';
+import { AxiosDataFetcher, AxiosDataFetcherConfig } from '../axios-fetcher';
+import { HttpDataFetcher, fetchData } from '../data-fetcher';
+import debug from '../debug';
 
-export interface LayoutService {
+/**
+ * Resolves layout service url
+ * @param {LayoutServiceConfig} [options] layout service options
+ * @param {string} verb to do
```

Comment:
I believe it's not a `verb`, because we are using `render` and `placeholder`, and `placeholder` doesn't fit it

Reply:
"verb" is the name of the parameter, and the "to do" is to figure out proper comments for it since I could not tell what this param is supposed to be based on looking at the function. sounds like you are familiar with the usage; can you let me know what the comment should be?

Reply:
@anastasiya29 May be it can be `action`? I'm looking at doc: https://jss.sitecore.com/docs/fundamentals/services/layout-service
`Using the Layout Service` - article
And **description**: Layout Service action

Reply:
Added more descriptive comment